shared-bindings/digitalio/DigitalInOut.c

```diff
@@ -65,7 +88,11 @@ STATIC mp_obj_t digitalio_digitalinout_make_new(const mp_obj_type_t *type,
     digitalio_digitalinout_obj_t *self = m_new_obj(digitalio_digitalinout_obj_t);
     self->base.type = &digitalio_digitalinout_type;
 
+    #if CIRCUITPY_CYW43
+    const mcu_pin_obj_t *pin = validate_obj_is_free_pin_including_cyw43(args[0]);
+    #else
     const mcu_pin_obj_t *pin = validate_obj_is_free_pin(args[0]);
+    #endif
```

Comment:
Maybe add a comment here that cyw43 only supports `DigitalInOut` pins (which is why it's being checked here)? I originally thought you could push this down into `validate_obj_is_free_pin()`, but that doesn't work because we don't want the cyw43 pins to be valid for anything but `DigitalInOut`.